### ISSUE TITLE
Error in Average Formula

### ISSUE THREAD

**Issue by Callindrill** [HendrickxJorn/Gobln.DiceBag#1]
I just wanted to say that there is an error in the average calculation. It should be:
```c#
/// <summary>
/// Avarage value of the dice
/// </summary>
public decimal Avarage { get { return ((decimal)Max + Min) / 2; } }
```
This expands out (and then  "simplifies") down to:
```c#
/// <summary>
/// Avarage value of the dice
/// </summary>
public decimal Avarage { get { return _amount * (decimal)(_size + 1) / 2 + _modifier; } }
```

**Comment by Callindrill** [comment]
Of course, if you want to save space on the casts, you can just do something like: 
```c#
_amount * (_size + 1) / 2.0M + _modifier;
```

**Comment by HendrickxJorn** [comment]
Thx for the bug
Its fixed in the version 1.0.2
